src/QuantizedMeshIO.cpp

```diff
@@ -295,6 +298,25 @@ static void write_qmheader(BinaryIO& bio,
     bio.write_double(qmheader.horizon_occlusion.z, e);
 }
 
+bool point_to_ecef(Vertex& p)
+{
+    OGRSpatialReference mercator;
+    OGRSpatialReference ecef;
+
+    mercator.importFromEPSG(3857);
+    ecef.importFromEPSG(4978);
```

Comment:
is it really necessary to allocate and initiate those SpatialReferences on every conversion of a point? Cant those be put on the class and just initialized once?

Reply:
This function is called once per tile, hence I don't think adding an extra class is worth it.